scripts/audio.js

```diff
@@ -61,9 +61,14 @@ H5P.Audio = (function ($) {
       'class': AUDIO_BUTTON + " " + PLAY_BUTTON
     }).appendTo(self.$inner)
       .click( function () {
+        if (!self.isEnabled) {
```

Comment:
self.isEnabled is not defined. I guess you meant self.isEnabled() ?

Reply:
Yup, sorry I missed that.